homeassistant/components/squeezebox/media_player.py

```diff
@@ -234,11 +212,24 @@ def __init__(self, player: Player, server: Server) -> None:
             identifiers={(DOMAIN, self._attr_unique_id)},
             name=player.name,
             connections={(CONNECTION_NETWORK_MAC, self._attr_unique_id)},
-            via_device=(DOMAIN, server.uuid),
             model=player.model,
             manufacturer=_manufacturer,
         )
 
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        if self._previous_media_position != self.media_position:
+            self._previous_media_position = self.media_position
+            self._last_update = utcnow()
+        self._attr_available = self._coordinator.available
+        self.async_write_ha_state()
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self._attr_available
```

Comment:
We should also take the super().available into account

Reply:
Yes, each of these is a necessary but not sufficient condition for the player actually being available.